aws/resource_aws_instance_test.go

```diff
@@ -494,6 +508,10 @@ func TestAccAWSInstance_blockDevices(t *testing.T) {
 				return fmt.Errorf("block device doesn't exist: /dev/sdd")
 			}
 
+			if _, ok := blockDevices["/dev/sde"]; !ok {
+				return fmt.Errorf("block device doesn't exist: /dev/sde")
+			}
```

Comment:
```suggestion
			if _, ok := blockDevices["/dev/sdf"]; !ok {
				return fmt.Errorf("block device doesn't exist: /dev/sdf")
			}
```

Reply:
This should fix the last remaining acceptance test failure.

Reply:
Aha, thanks!